src/react/ui-elements/__tests__/Activity.test.jsx

```diff
@@ -0,0 +1,34 @@
+import Activity from '../Activity';
+import { List } from 'immutable';
+import React from 'react';
+import { reduxMount } from '../../utils/test';
+
+describe('Activity', () => {
+    const TEST_MESSAGE = 'This is a test message';
+
+    it('Activity should render with a custom message', () => {
+        const { component } = reduxMount(<Activity/>, {
+            networkActivity: { counter: 1, messages: List([TEST_MESSAGE]) },
+        });
+
+        expect(component.find(Activity).isEmptyRender()).toBe(false);
+        expect(component.find('#activity-message').children().at(2).text()).toBe(TEST_MESSAGE);
+    });
+
+    it('Activity should render with a default message', () => {
+        const { component } = reduxMount(<Activity/>, {
+            networkActivity: { counter: 2, messages: List() },
+        });
+
+        expect(component.find(Activity).isEmptyRender()).toBe(false);
+        expect(component.find('#activity-message').children().at(2).text()).toBe('Working...');
```

Comment:
Can you please export/import the default message?

Reply:
done